unittest/include_gunit.h

```diff
@@ -17,7 +17,7 @@
 #include "fileio.h"   // for tesseract::File
 #include "gtest/gtest.h"
 
-const char* FLAGS_test_tmpdir = ".";
+const char* FLAGS_test_tmpdir = "./tmp";
```

Comment:
That directory is missing for builds which are not started in the root directory, so a lot of tests fail or crash currently. Do we need this change?

Reply:
When I was running `make check` in tesseract root directory all files generated by unittests were being created in the unittest root directory. `make clean` did not remove them. So I thought it would be helpful to have a separate directory for the generated files. 

There maybe a better way to accomplish this. Please change as you see fit. Thanks.